client/express-checkout/index.js

```diff
@@ -590,7 +543,8 @@ jQuery( ( $ ) => {
 				} = wcpayECEPayForOrderParams;
 
 				if ( total === 0 ) {
-					wcpayECE.hide();
+					expressCheckoutButtonUi.hide();
+					expressCheckoutButtonUi.getButtonSeparator().hide();
```

Comment:
Before, the `hide()` method was calling `hide()` also on the separator. But the `show()` method wasn't.

I wanted to bring consistency here, so now the `hide()` and `show()` method won't call `hide()` or `show()` on the separator.